examples/javascript/PitchDetection/PitchDetection_Game/sketch.js

```diff
@@ -112,7 +112,7 @@ function draw() {
 }
 
 function dist(x1, y1, x2, y2) {
-  return Math.sqrt(Math.exp((x2 - x1), 2) + Math.exp((y2 - y1), 2));
+  return Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));
```

Comment:
For the sake of our linting rules, can we use our exponentiation operator `**` instead of `Math.pow`?
```
Math.sqrt(((x2 - x1) ** 2) + ((y2 - y1) ** 2));
```

Reply:
+1 ^^